pygenesys/tests/test_eia_data.py

```diff
@@ -0,0 +1,204 @@
+from pygenesys.data.eia_data import *
```

Comment:
Might be better to do an `import pygenesys.data.eia_data as ...` instead importing everything, so that when data/classes/functions in `pygenesys.data.eia_data` are referenced we know where they are coming from.

Reply:
That's valid, though I'm not sure it clarifies very much since this file is specifically for testing functions in `eia_data.py`. In most other circumstances, I would agree.